index.html

```diff
@@ -45,6 +45,180 @@ <h1 class="headline">
+                </p>
+                
+                <div class="social-handles">
+                    <p>Let's Connect</p>
+                    <ul class="social-icons">
+                        <li class="github"><a href="https://github.com/ishpaul777"><img src="./assets/icons/github.svg" alt="github"></a></li>
+                        <li class="linkedin"><a href="https://www.linkedin.com/in/ishpaul-singh-264590226/"><img src="./assets/icons/linkedin.svg" alt="linkedin"></a></li>
+                        <li class="twitter"><a href="https://twitter.com/ishpaul_777"><img src="./assets/icons/twitter.svg" alt="twitter"></a></li>
+                        <li class="instagram"><a href="https://www.instagram.com/ishpaul_777"><img src="./assets/icons/instagram.svg" alt="instagram"></a></li>
+                    </ul>
+                </div>
+                <button type="button" class="resume-btn">Get my resume</button>
+                <div class="skills-info">
+                    
+                    <div class="languages">
+                        <div class="skill-type">
+                            <p>Languages</p>
+                            <a href="#">
+                                <img src="./assets/icons/chevron-down.svg" alt="down-arrow">
+                            </a>
+                        </div>
+                        <div>
+                            <div class="lang-skill">
+                                <img src="./assets/icons/javascript-logo.png" alt="javascript-logo">
+                                <p>Javascript</p>
+                            </div>
+                            <div class="lang-skill">
+                                <img src="./assets/icons/html-logo.png" alt="html-logo">
+                                <p>HTML</p>
+                            </div>
+                            <div class="lang-skill">
+                                <img src="./assets/icons/css-logo.png" alt="css-logo">
+                                <p>CSS</p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="frameworks">
+                        <div class="skill-type">
+                            <p>Frameworks</p>
+                            <a href="#">
+                                <img src="./assets/icons/chevron-right.svg" alt="down-arrow">
+                            </a>
+                        </div>
+                    </div><hr>
+                    <div class="skills">
+                        <div class="skill-type">
+                            <p>Skills</p>
+                            <a href="#">
+                                <img src="./assets/icons/chevron-right.svg" alt="down-arrow">
+                            </a>
+                        </div><hr>
+                    </div>
+                    
+                </div>
```

Comment:
the code will become mess

Reply:
@ShadyShawkat  I cant do this the code will become mess and hard to understand even for me.

Reply:
Hi @ishpaul777, this is a good practice, to put a list of things that relate to one another inside a `ul` tag. Since there's a list of skills, you can wrap them with a `ul` tag which is very understandable. Also, since the first skill ("Languages") has multiple points (like the list of languages) this also gets to be a `ul`. 

I get that nested `ul` tags can look confusing in the beginning, but the truth is that it's way more convenient and better than just regular nested `div` tags. You can read more about nested lists on [this slackoverflow thread](https://stackoverflow.com/questions/5899337/proper-way-to-make-html-nested-list#:~:text=Nested%20lists%20often%20form%20the,to%20nest%20lists%20too%20deeply.). 

I hope this helps.

Reply:
@ShadyShawkat I still thinks my code is more readable and easy.
doing this makes things turn messy and unreadable